tests/phpunit/ConnectionTest.php

```diff
@@ -46,19 +62,19 @@ public function testQueryTimeout(): void
 
         // long-running query
         $connection->executeQuery(
-            'WITH RECURSIVE counter AS (
-                      SELECT 1 AS n
-                      UNION ALL
-                      SELECT n+1 FROM counter WHERE n < 10000
-                    )
-                    
-                    SELECT 
-                      a.n AS val1, 
-                      b.n AS val2
-                    FROM 
-                      counter a 
-                    CROSS JOIN 
-                      counter b;'
+            self::TIMEOUT_RECURSIVE_QUERY
+        );
+    }
+
+    public function testRecursiveQueryWithoutTimeout(): void
```

Comment:
@zajca podle me by to chtelo i test kde bude "dlouha" query ktera bude ale success.

Reply:
Ty jo už hodinu studuju jak udělat nějaků dlůhů query v BQ a nic teda, zkusil sem UDF a udělat sleep přes JS, ale ta fce mě nikdy nedoběhne :D 

```
CREATE OR REPLACE FUNCTION TEST_Sleep()
 RETURNS INT64
LANGUAGE js AS r"""
  let waitUntil = new Date().getTime() + 1000;
  while(new Date().getTime() < waitUntil){
  }
  return 0;
  """;

SELECT TEST_Sleep();
```

Reply:
hmm, task asi to neresme ted a hodit na to nejake issue?

Reply:
Mozna by stacil akceptacni test https://keboola.atlassian.net/browse/GCP-446